### DISCUSSION TITLE
gpth-windoza.exe doesn't do anything ...

### DISCUSSION THREAD

**Discussion by jojo-gfx** [TheLastGimbus/GooglePhotosTakeoutHelper#279] [I NEED HELP]
Hi,
would try gpth-windoza.exe and it starts in a CMD window, but it stops and need a folder.
So far yes, but how? Insert a path there doesn't work if I hit Enter. Generally can't do anything there!?
Anyone an idea what I have to do there? Also no request is pop up there or anything...

Greetings

**Comment by jojo-gfx** [comment]
Ah, ok, have it, ... has to start it as administrator.

But, then all config given it seems to do anything but zero output then. :(

**Comment by jojo-gfx** [comment]
I do it per hand now...

**Comment by TheLastGimbus** [comment]
You can look at #265 , it may help you for not - otherwise i don't know what's your problem :shrug:
